agg.pretty/PIZERO.kicad_mod

```diff
@@ -0,0 +1,78 @@
+  (pad 25 thru_hole circle (at 6.35 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 24 thru_hole circle (at 3.81 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 23 thru_hole circle (at 3.81 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 22 thru_hole circle (at 1.27 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 21 thru_hole circle (at 1.27 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 20 thru_hole circle (at -1.27 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 19 thru_hole circle (at -1.27 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 18 thru_hole circle (at -3.81 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 17 thru_hole circle (at -3.81 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 16 thru_hole circle (at -6.35 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 15 thru_hole circle (at -6.35 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 14 thru_hole circle (at -8.89 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 13 thru_hole circle (at -8.89 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 12 thru_hole circle (at -11.43 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 11 thru_hole circle (at -11.43 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 10 thru_hole circle (at -13.97 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 9 thru_hole circle (at -13.97 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 8 thru_hole circle (at -16.51 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 7 thru_hole circle (at -16.51 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 6 thru_hole circle (at -19.05 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 5 thru_hole circle (at -19.05 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 4 thru_hole circle (at -21.59 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 3 thru_hole circle (at -21.59 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 2 thru_hole circle (at -24.13 -13.17) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad 1 thru_hole rect (at -24.13 -10.63) (size 1.9 1.9) (drill 1) (layers *.Cu *.Mask F.SilkS))
+  (pad "" np_thru_hole circle (at -29 -11.9) (size 2.75 2.75) (drill 2.75) (layers *.Cu *.Mask))
+  (pad "" np_thru_hole circle (at 29 -11.9) (size 2.75 2.75) (drill 2.75) (layers *.Cu *.Mask))
+  (pad "" np_thru_hole circle (at 29 11.1) (size 2.75 2.75) (drill 2.75) (layers *.Cu *.Mask))
+  (pad "" np_thru_hole circle (at -29 11.1) (size 2.75 2.75) (drill 2.75) (layers *.Cu *.Mask))
```

Comment:
the part centre should be the geometric centre, so the holes should be at +-11.5 instead of +11.1 and -11.9 (this will also affect the offset of the 3d model below)